app/services/scenes/scene.ts

```diff
@@ -306,18 +306,19 @@ export class Scene {
   private reconcileNodeOrderWithObs() {
     const obsScene = this.getObsScene();
     const destOrder = this.getItems().map(item => item.obsSceneItemId);
-    const currentOrder = this.getObsScene()
+    const currentOBSOrder = obsScene
       .getItems()
       .reverse()
       .map(item => item.id);
 
-    destOrder.forEach(ind => {
-      if (destOrder[ind] === currentOrder[ind]) return;
-      const itemToMoveInd = currentOrder.indexOf(destOrder[ind]);
-      const itemToMove = currentOrder[itemToMoveInd];
-      currentOrder.splice(itemToMoveInd, 1);
-      currentOrder.splice(ind, 0, itemToMove);
-      obsScene.moveItem(itemToMoveInd, ind);
+    destOrder.forEach(itemId => {
+      const itemToMoveInd = currentOBSOrder.indexOf(itemId);
+      const currentItemInd = destOrder.indexOf(itemId);
+      if (itemToMoveInd === currentItemInd) return;
```

Comment:
Do you know any steps to reproduce this issue from UI?

Reply:
no. our crash reports not showing any prior user actions that can caused this.